src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -82,7 +82,13 @@ pub struct CloudwatchLogsSinkConfig {
 }
 
 inventory::submit! {
-    SinkDescription::new_without_default::<CloudwatchLogsSinkConfig>("aws_cloudwatch_logs")
+    SinkDescription::new::<CloudwatchLogsSinkConfig>("aws_cloudwatch_logs")
+}
+
+impl GenerateConfig for CloudwatchLogsSinkConfig {
+    fn generate_config() -> toml::Value {
+        toml::Value::Table(Default::default())
```

Comment:
What does the `Default::default` evaluate to in this case? Might it be an empty table, rather than the default `GenerateConfig`?

I don't see a test in this file, but - we probably should test this too.

Reply:
This is `BTreeMap::default()`, a test for this will fail, most of the sinks don't generate a valid configuration at the moment, adding all of them in a PR will not be practical, so I use this empty value at the moment, and fixing them later.